Get user Lat/Long in details.jsp

fstelfox opened the pull request.

fstelfox committed:
Get user Lat/Long in details.jsp

HCHegarty committed:
Merge branch 'master' into GetUserLatLng

HCHegarty reviewed (approved):
Mostly just changing values or additions, removed one conflict

HCHegarty committed:
Changed values of slider starting values

Changed the value of "maxRange" from 300000 to 150000 & value of "rangeRange" from 3000 to 300
